crates/sp-domains/src/lib.rs

```diff
@@ -1532,9 +1532,6 @@ sp_api::decl_runtime_apis! {
         /// Returns the current epoch and the next epoch operators of the given domain
         fn domain_operators(domain_id: DomainId) -> Option<(BTreeMap<OperatorId, Balance>, Vec<OperatorId>)>;
 
-        /// Get operator id by signing key
-        fn operator_id_by_signing_key(signing_key: OperatorPublicKey) -> Option<OperatorId>;
```

Comment:
I'm actually surprised this is only used in malicious operator and not the actual one.
maybe a remnent from the bundle equivocation feature we forgot to remove I guess.